pkgs/build-support/node/build-npm-package/hooks/npm-install-hook.sh

```diff
@@ -25,8 +25,7 @@ npmInstallHook() {
         else "invalid type " + $typ | halt_error end' "${npmWorkspace-.}/package.json")
 
     while IFS= read -r man; do
-        mkdir -p "$out/share/man"
-        ln -s "$packageOut/$man" "$out/share/man"
+        installManPage "$packageOut/$man"
```

Comment:
This isn't quite accurate either. According to https://docs.npmjs.com/cli/v9/configuring-npm/package-json/#man we're supposed to modify the name of the man page. That would be easiest to do if we modify `installManPage` to accept the man page name as an optional argument.

Reply:
TIL about "installManPage", thanks! No objection to leaving that bug for now, it's still better than the previous or current status and I don't want to block this particular PR on the man page question.